service/grails-app/services/org/olf/rs/ProtocolMessageBuildingService.groovy

```diff
@@ -130,19 +130,33 @@ class ProtocolMessageBuildingService {
      * AvailabilityNote
     ]
      */
-    message.requestedDeliveryInfo = [
-      // SortOrder
-      address:[
-        physicalAddress:[
-          line1:req.pickupLocation,
-          line2:null,
-          locality:null,
-          postalCode:null,
-          region:null,
-          county:null
+
+    // Since ISO18626-2017 doesn't yet offer DeliveryMethod here we encode it as an ElectronicAddressType
+    if (req.deliveryMethod) {
```

Comment:
As you have Courier as a delivery method you may want that to drop through to the else path so it goes as a physical address

Reply:
Thanks, good point. Changed it to only use ElectronicAddressType for a deliveryMethod of url rather than try to slot it into the physical address somewhere as that'd be even more of a stretch I imagine.